From 1bf4ea07eafee0f83a52bce0b732211789af1eac Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Thu, 8 Jul 2021 23:16:04 +0200
Subject: [PATCH] uriparser: fix CMake compilation

Fixes:
ninja: error: loading 'build.ninja': No such file or directory

Source:
https://github.com/openwrt/packages/pull/15012/files#diff-03f422c03a3e924a3ff829ea75c637e7304a33ef39d451c5aec90127f11d0635R20
---
 libs/uriparser/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libs/uriparser/Makefile b/libs/uriparser/Makefile
index 0b0879b2e..62d8c87bb 100644
--- a/libs/uriparser/Makefile
+++ b/libs/uriparser/Makefile
@@ -47,7 +47,7 @@ define package/liburiparser/description
   strictly rfc 3986 compliant uri parsing and handling library written in c89
 endef
 
-CMAKE_OPTIONS := \
+CMAKE_OPTIONS += \
 	-DBUILD_SHARED_LIBS=ON \
 	-DURIPARSER_BUILD_CHAR=ON \
 	-DURIPARSER_BUILD_DOCS=OFF \
-- 
2.37.1

